tests/unit/s2n_send_multirecord_test.c

```diff
@@ -13,24 +13,38 @@
  * permissions and limitations under the License.
  */
 
-#include <sys/param.h>
 #include <math.h>
 #include <pthread.h>
+#include <sys/param.h>
 
+#include "api/s2n.h"
 #include "s2n_test.h"
 #include "testlib/s2n_testlib.h"
-
-#include "api/s2n.h"
 #include "tls/s2n_post_handshake.h"
-#include "tls/s2n_tls.h"
 #include "tls/s2n_record.h"
+#include "tls/s2n_tls.h"
 #include "utils/s2n_random.h"
 
-#define CLOSED_SEND_RESULT { .result = -1, .error = EPIPE }
-#define BLOCK_SEND_RESULT { .result = -1, .error = EAGAIN }
-#define PARTIAL_SEND_RESULT(bytes) { .result = bytes, .error = EAGAIN }
-#define EXPECTED_SEND_RESULT(bytes) { .result = bytes, .assert_result = true }
-#define OK_SEND_RESULT { .result = INT_MAX }
+#define CLOSED_SEND_RESULT           \
+    {                                \
```

Comment:
Wow, I really hate this formatting. Might be worth turning clang-format off for this block. Same for similar defines in s2n_send_test.